lms/templates/admin/base.html.jinja2

```diff
@@ -6,7 +6,7 @@
     <title>
          {% block title %}
             {% if self.header() %}
-                {{ self.header() }} - LMS Admin
+                {{ self.header()|striptags }} - LMS Admin
```

Comment:
If not title is defined we try to use the whole header block of the page, strip any HTML while doing that.